src/javascript/gameInstance.js

```diff
@@ -283,20 +292,20 @@ export default class GameInstance {
     ) {
       this.timeArr[this.lastAddedIdx].k += key;
     } else {
-      // this.timeArr.push({
-      //   t: Number(cTime.toFixed(3)),
-      //   k: key,
-      // });
-      // this.timeArrIdx = this.timeArr.length - 1;
-      // add at idx
-      this.timeArr.splice(this.timeArrIdx, 0, {
+      this.timeArr.push({
         t: Number(cTime.toFixed(3)),
         k: key,
       });
+      // this.timeArrIdx = this.timeArr.length - 1;
+      // add at idx
+      // this.timeArr.splice(this.timeArrIdx, 0, {
+      //   t: Number(cTime.toFixed(3)),
+      //   k: key,
+      // });
```

Comment:
Remove commented-out code if not needed.

Reply:
Yeah, I was deciding this and another way, I'll keep it there for now, will remove once this is settled.